### PR TITLE
DIFM Lite: Revamp DIFM Lite flow

### PR DESCRIPTION
**Pull request by aneeshd16**
### Changes proposed in this Pull Request

Context: pdh1Xd-iQ-p2

This PR will act as the parent PR and should be merged when the following PRs are completed and merged. Please do not commit directly to this branch.

- [x] New redux state for site info form (7171a59235e819377c428fb1fabe13b7de6f448c)
- [x] Site Information form step  #59986
- [x] Website content step  
    - [x] #60293
    - [x] #60436
    - [x] Step UI shell design Implementation     - #60548
    - [x] Calypso redux Data integration with step UI     - #60548
    - [x] Endpoint for website content persistence D74241-code
    - [x] Initialize the 4th page from the site category. https://github.com/Automattic/wp-calypso/pull/60998
    - [x] #61121
    - [x] Overall flow integration and completion https://github.com/Automattic/wp-calypso/pull/61029
- [x] Remove Typeform #60029 (blocked by #59986)
- [x] Redesigned New/Existing site and Site Picker steps #60479
- [x] Post-checkout screen
  - [x] User-action required https://github.com/Automattic/wp-calypso/pull/61070
  - [x] No user-action required: https://github.com/Automattic/wp-calypso/pull/60881

### Bug fixes
- [x] #61309
- [x] https://github.com/Automattic/wp-calypso/pull/60957
- [x] #61156

#### Testing instructions

pdh1Xd-xv-p2

#### Screenshots for translators

image
image
image
image
image
image
image

### CODE REVIEW

File: client/signup/steps/site-info-collection/form-components.tsx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+			<TextInputField
+				{ ...displayPhoneProps }
+				label={ translate( 'Phone Number' ) }
+				placeholder="+1 212 555 55 55"
+				onChange={ onChange }
+			/>
+			<AddressField>
+				<TextAreaField
+					{ ...displayAddressProps }
+					label={ translate( 'Physical address' ) }
```
Comment (a8ci18n) [inline client/signup/steps/site-info-collection/form-components.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Must be a physical address.' )` *ES Score: 7*

File: client/signup/accordion-form/form-components.tsx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+			<TextInputField
+				{ ...displayPhoneProps }
+				label={ translate( 'Phone Number' ) }
+				placeholder="+1 212 555 55 55"
+				onChange={ onChange }
+			/>
+			<AddressField>
+				<TextAreaField
+					{ ...displayAddressProps }
+					label={ translate( 'Physical address' ) }
```
Comment (a8ci18n) [inline client/signup/accordion-form/form-components.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Must be a physical address.' )` *ES Score: 7*

File: client/signup/steps/website-content/wordpress-media-upload.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+			return (
+				<FilePicker key={ mediaId } accept="image/*" onPick={ onPick }>
+					<FileSelectThumbnailContainer>
+						<div>
+							<Preview>
+								<CroppedImage>
+									<img src={ uploadedImageUrl } alt={ imageCaption } />
+								</CroppedImage>
+							</Preview>
+							<Label>{ translate( 'Change File' ) }</Label>
```
Comment (a8ci18n) [inline client/signup/steps/website-content/wordpress-media-upload.tsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **133** times:
`translate( 'Change file' )` *ES Score: 8*

File: client/signup/steps/new-or-existing-site/index.tsx [hunk 4]
```diff
@@ -18,8 +21,10 @@ export default function NewOrExistingSiteStep( props: Props ): React.ReactNode {
 	const dispatch = useDispatch();
 	const translate = useTranslate();
 
-	const headerText = translate( 'Choose where you want us to build your site.' );
-	const subHeaderText = '';
+	const headerText = translate( 'Do it For Me' );
```
Comment (a8ci18n) [inline client/signup/steps/new-or-existing-site/index.tsx:24]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Do It For Me (Lite)' )` *ES Score: 7*

**Review by ddc22 — APPROVED** [review]
Confidence checks passed! LGTM! 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7149428

Thank you @aneeshd16 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7149428).

**Pull request merged**